doc/rfd/rfd/0007/README.md

```diff
@@ -0,0 +1,47 @@
+---
+authors: Jaime Gomes <jaime.gomes@suse.com>
+state: draft
+discussion: 
+---
+
+# RFD 7 Deployment and Upgrade Benchmark
+
+## Introduction
+
+### Motivation
+
+Currently, we do not have any data that can corroborate with some eye evidence that deployment times
+of KubeCF are increasing between releases.
+
+### Goal
+
+Be able to measure and collect time metrics during a fresh deployment and between upgrades, on each
+component (pod) present on KubeCF.
+
+With the time data collected, both upstream projects and downstream product teams can investigate
+if there are optimizations that can lead to a better and more faster deployments that will support a
```

Comment:
```suggestion
if there are optimizations that can lead to better and faster deployments that will support a
```

The `a better` is singular, the `deployments` is plural. Removing the `a` to match that plural.
`faster` itself implies `more`.